ground/gcs/src/libs/tlmapcontrol/mapwidget/tlmapwidget.h

```diff
@@ -45,6 +45,7 @@
 #include "mapcircle.h"
 #include "waypointcurve.h"
 #include "waypointitem.h"
+#include "../core/corecommon.h"
```

Comment:
This refers to the core part of the library so it is independent of the repo layout. I can add this path to the INCLUDEPATH var if you prefer.